gox: add page (#3004)
tldr-pages/tldr: pages/common/gox.md

# gox

> > A tool for Go cross compilation.

Reviewer: If I understand it right, maybe this would read better?

```suggestion
> A tool for cross-compiling Go programs.
```

> > A tool for cross-compiling Go programs.

Reviewer 2: Please add the home page url in a new line after this.

Reviewer: Like this:

```markdown
> Homepage: <https://example.com>.
```

> - Compile current Go directory for all OS and arch:

Reviewer: I assume Go programs are laid out in a specific directory structure?

Perhaps this will read better?
```suggestion
- Compile Go program in the current directory for all operating systems and architecture combinations:
```

`gox`

> - Compile from URL:

Reviewer: Perhaps this will read better?
```suggestion
- Download and compile a Go program from a remote URL:
```

`gox {{url_1}} {{url_2}}`

> - Compile current directory for `{{os}}`:

Reviewer: Perhaps this will read better?
```suggestion
- Compile current directory for a particular operating system:
```

> `gox -os="{{os}}"`

Reviewer: Are you missing an extra dash there?

Author: Negative. As per the `gox --help` a single dash is listed, but both seem to play nice. I added single dashes because that's how it's displayed in the help.

> - Compile current directory for `{{os}}` running `{{arch}}`:

Reviewer: Perhaps this will read better?
```suggestion
- Compile current directory for a single operating system and architecture combination:
```
